homeassistant/components/freebox/number.py

```diff
@@ -0,0 +1,130 @@
+"""Support for Freebox Delta, Revolution and Mini 4K."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+
+from freebox_api.exceptions import InsufficientPermissionsError
+
+from homeassistant.components.number import (
+    NumberEntity,
+    NumberEntityDescription,
+    NumberMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .router import FreeboxRouter
+
+
+@dataclass
+class FreeboxNumberSetValueMixin:
+    """Mixin for required keys."""
+
+    async_set_value: Callable[[FreeboxRouter, int], Awaitable]
+
+
+@dataclass
+class FreeboxNumberEntityDescription(
+    NumberEntityDescription, FreeboxNumberSetValueMixin
+):
+    """Class describing Freebox number entities."""
+
+
+LCD_NUMBER_DESCRIPTIONS: tuple[FreeboxNumberEntityDescription, ...] = (
+    FreeboxNumberEntityDescription(
+        key="lcd_brightness",
+        name="LCD Brightness",
```

Comment:
When the `_attr_has_entity_name` is set (as suggest in another comment), we can remove this name and add it to the `strings.json` translations file (making this translatable).